cprover_bindings/src/goto_program/symtab_transformer/transformer.rs

```diff
@@ -153,6 +156,19 @@ pub trait Transformer: Sized {
         transformed_typ.infinite_array_of()
     }
 
+    fn transform_type_mathematical_function(
+        &mut self,
+        domain: &Vec<Type>,
+        codomain: &Type,
+    ) -> Type {
+        let transformed_domain: Vec<Type> = domain.iter().map(|x| self.transform_type(x)).collect();
```

Comment:
Do you need the explicit type annotation here?

Reply:
Removed explicit type annotation after changing type of domain to &[Type].